client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -17,95 +17,92 @@ export const MAX_POSTS_TO_SHOW = 6;
 
 function PlaceholderPostEdit( props ) {
 	return (
-		<div
+		<ul
```

Comment:
Noting that since different themes introduce padding/margin to `ul` and `li` elements, would be good to make sure we reset them so not to have surprises.

Reply:
There is an `!important` statement in the Sass code that should cover this.

Reply:
Could we increase specifity in some other way than `!important` which is often a difficult pattern otherwise (https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)? We just had a similar case with slideshow block https://github.com/Automattic/wp-calypso/pull/31019

https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#general-syntax-and-writing-rules